### PR TITLE
add endpoint update /:userId

### PR DESCRIPTION
**Pull request by Andrij-Maksymyshyn**

### CODE REVIEW

File: api/users/schemas.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    })
+    .required(),
+
+  body: Joi.object().keys({
+    fullName: Joi.string()
+      .alphanum()
+      .min(4)
+      .max(25)
+      .trim()
+      .error(new Error("'fullName' is not valid")),
```
Comment (chestnyh) [inline api/users/schemas.js]: You have the same code in `newUserSchema`. You should avoid such code duplication. Try to find a solution when these objects shares some parts of code.

File: api/users/schemas.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    email: Joi.string()
+      .regex(EMAIL_REGEXP)
+      .required()
+      .lowercase()
+      .trim()
+      .error(new Error("Email is not valid")),
```
Comment (chestnyh) [inline api/users/schemas.js]: The same as here https://github.com/Andrij-Maksymyshyn/blog/pull/11/files#r1158171613

**Review by chestnyh — APPROVED** [review]

File: api/users/schemas.js (hunk 1 above)
Comment (Andrij-Maksymyshyn) [reply api/users/schemas.js]: fixed

File: api/users/schemas.js (hunk 2 above)
Comment (Andrij-Maksymyshyn) [reply api/users/schemas.js]: fixed

**Pull request merged**